### PR TITLE
ADH Branding

### PR DESCRIPTION
**Pull request by anton-dahlin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Authorization Code Flow + PKCE DotNet Sample and Test
 
+| :loudspeaker: **Notice**: Samples have been updated to reflect that they work on AVEVA Data Hub. The samples also work on OSIsoft Cloud Services unless otherwise noted. |
+| -----------------------------------------------------------------------------------------------|  
+
 **Version:** 1.2.23
```
Comment (derekendres) [inline README.md:6]: This one i would bump the version since we are renaming functions.  Probably 1.3.0

Comment (derekendres) [reply README.md:6]: And history file

**Review by derekendres — CHANGES_REQUESTED** [review]
version and history here

File: README.md (hunk 1 above)
Comment (anton-dahlin) [reply README.md:6]: great point, I will do that for the others that follow this behavior as well. Thanks!

**Review by derekendres — APPROVED** [review]

**Review by kelseyduffy — APPROVED** [review]

**Pull request merged**